knowledge_base/Package-Scripts.md

```diff
@@ -473,6 +474,12 @@ Definition: `tsc --noEmit`
 
 Description: Runs a compilation of TypeScript files based on tsconfig.json; does not emit files.
 
+### sanity
+
+Definition: `chmod u+x scripts/sanity.sh && ./scripts/sanity.sh`
+
+Description: Sanity scripts developers should run locally before pushing code
```

Comment:
```suggestion
Description: Sanity scripts developers should run locally before pushing code, comprising a linter, a check-types, and unit tests. Must be run from root.
```

I agree we should move towards running scripts from root, but in the meantime might be worth explicitly noting which scripts must be run root